### DISCUSSION TITLE
Struggling to find correct data

### DISCUSSION THREAD

**Discussion by PatrickSteiner** [tijsverkoyen/HomeAssistant-FusionSolar#161] [Q&A] [answered]
Hi community,

I'm struggling to find the right entities to use in my own dashboard. All I want is to get those 4 values from the Fusion100 Dashboard

![image](https://github.com/tijsverkoyen/HomeAssistant-FusionSolar/assets/4146860/fc64e0ba-38d5-4ae6-b885-f0ff38fc5fe0)

What is everyone using for
 - power coming from "the roof" ( sensor.meter_1_realtime_power ? )
 - power going to the grid ( ?? )
 - current house consumption ( ?? )
 
 What is everyone using?

**Comment by fvillemin** [comment, marked as answer]
Hello,
I'm not 100% sure of the values but it seems quite good when compared to FusionSolar
In my Home Assistant, i'm using :

- power coming from "the roof" : sensor.inv_hvxxxxxx_mppt_total_input_power
- power going to the grid : sensor.meter_1_realtime_power    (positive = i send to the grid, negative = i buy from the grid)
- battery : sensor.battery_a_charge_discharge_power  (positive = charging battery, negative = using battery)
- current house consumption : this one is not given by the installation. For me i think a template like that :

{{states.sensor.inv_hvxxxxxx_mppt_total_input_power.state | float - states.sensor.battery_1_charge_discharge_power.state |float - states.sensor.meter_1_realtime_power.state | float  }}

to be clear : production - battery - meter_1_power

It's not very accurate because Home Assistant doesn't take all values at the same time, but it is the best way i have found

**Comment by PatrickSteiner** [comment]
Thanks a lot for this very helpful answer! Will change my dashboard based on your recommendations.
